docker-compose.yml

```diff
@@ -730,16 +664,10 @@ services:
 ################################################################################
 # DELTA SUBSIDIES: START
 ################################################################################
-  delta-producer-background-jobs-initiator-subsidies:
+  delta-producer-background-jobs-initiator-generic:
```

Comment:
The docker-compose.dev.yml file should also be updated, to add the `restart: "no"` on the generic initiator and remove the specific job initiators